src/seq/minhash.jl

```diff
@@ -18,12 +18,12 @@ hashes for kmers of a given length. The type contains two parameters:
 """
 immutable MinHashSketch
     sketch::Vector{UInt64}
-    kmersize::Int
+    kmersize::Integer
```

Comment:
I think I see, but just to be sure, you're saying line 21 there should be `Int`, but the constructor on line 23 should be `Integer`, yes?

So as a general rule, type fields should be specific where possible, but function arguments should be more permissive?